experimental/uefi/client/src/main.rs

```diff
@@ -14,18 +14,13 @@
 // limitations under the License.
 //
 
+use anyhow::Context;
+use grpc_unary_attestation::client::AttestationClient;
```

Comment:
Because of the prior refactor of https://github.com/project-oak/oak/pull/2764, we can simply drop in the existing handshake client & reuse it's logic.